lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -108,7 +104,7 @@ def instructor_dashboard_2(request, course_id):
         'studio_url': get_studio_url(course, 'course'),
         'sections': sections,
         'disable_buttons': disable_buttons,
-        'analytics_dashboard_message': analytics_dashboard_message
+        'analytics_dashboard_url': analytics_dashboard_url,
```

Comment:
I don't think that's ok, because we want people to be able to use the existing docs from edx.org as much as makes sense, and all of them refer to that tab as Analytics. (I agree it's not a great name, but I think we should stick w/ the status quo.)